framework/console/Controller.php

```diff
@@ -140,6 +140,10 @@ public function bindActionParams($action, $params)
             throw new Exception(Yii::t('yii', 'Missing required arguments: {params}', ['params' => implode(', ', $missing)]));
         }
 
+        // #9823
```

Comment:
No need to reference it since the change wasn't released yet and this is regression fix.